adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -115,6 +115,19 @@
                 if(initOptions.redirectUri) {
                     kc.redirectUri = initOptions.redirectUri;
                 }
+
+                if (initOptions.pkceMethod) {
+                    switch(initOptions.pkceMethod) {
+                        case "plain":
```

Comment:
btw. other libaries, e.g. [openid/AppAuth-JS](https://github.com/openid/AppAuth-JS/blob/a8c2962d7a24010ec1b4eb0e6eb3a9b0234f5733/src/authorization_request.ts#L96) explicitly don't support the pkce method `plain`.